recipes/server_install.rb

```diff
@@ -182,6 +188,17 @@
   notifies (node['rundeck']['restart_on_config_change'] ? :restart : :nothing), 'service[rundeck]', :delayed
 end
 
+template "/etc/init/rundeckd.conf" do
```

Comment:
does this fix the ubuntu problems we've been having? if so, take the ubuntu tests out of the acceptable failures in the .travis.yml